net/unbound/files/rootzone.sh

```diff
@@ -21,26 +21,35 @@
 ##############################################################################
 
 rootzone_uci() {
-  # TODO: Just structure to real UCI coming soon.
-  echo
+  local cfg=$1
+  
+  # This will likely be called outside of "start_service()" context
+  config_get_bool UNBOUND_B_DNSSEC "$cfg" validator 0
+  config_get_bool UNBOUND_B_NTP_BOOT "$cfg" validator_ntp 1
+  config_get UNBOUND_N_ROOT_AGE "$cfg" root_age 30
 }
 
 ##############################################################################
 
 roothints_update() {
-  # TODO: Maybe this will not be implemented.
-  echo
+  # TODO: Might not be implemented. Unbound doesn't natively update hints.
+  # Unbound philosophy is built in root hints are good for machine life.
+  return 0
 }
 
 ##############################################################################
 
 rootkey_update() {
   local basekey_date rootkey_date rootkey_age filestuff
-
-  # TODO: Just structure to real UCI coming soon.
+  
```

Comment:
That seems pretty common for "smart indent" enabled in any editor. Blank lines are left ready at the respective indent level.